### PR TITLE
Modify the default QFieldCloud server in QField

### PR DESCRIPTION
**Pull request by SeqLaz**
Added the section to modify the default QFieldCloud server in QField Sync and QField.

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-2878 Changing QFieldCloud instance from default app.qfield.cloud in QField and QField Sync](https://app.clickup.com/t/2192114/QF-2878)

**Comment by SeqLaz** [comment]
Hi @lucienicolier, @suricactus I added the section of modify the default QFieldCloud server in QField Sync and QField

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 1]
```diff
@@ -90,18 +85,33 @@ By double-clicking on a project in the list, you can see and edit the specific p
...
+
+QField and QField Sync connect to the official [QFieldCloud](https://app.qfield.cloud/). This default server can be modified to connect to a different server if needed.
+
+To modify the default server, follow these steps:
+
+1. Open the login screen in QField or QField Sync.
+2. Double-tap on the Nyuki icon.
+3. This action will reveal a field where you can enter the preferred server information.
+4. Enter the details of the desired server in the provided field.
+(Leave empty to connect to default server)
```
Comment (suricactus) [inline documentation/get-started/tutorials/get-started-qfc.en.md:98]:
```suggestion
(Leaving the field empty will connect to the default QFieldCloud server at app.qfield.cloud.)
```

**Review by suricactus — APPROVED** [review]
Thanks a lot!

Two comments: 
- it is QFieldSync (without space)
- avoid changing more than one thing in a single commit (and ideally single PR). In this case there are some whitespaces around.

No need to rewrite git history now, in the docs is okayish.

**Comment by SeqLaz** [comment]
Thank you a lot @suricactus, I'll also take those recommendation for future documentations

**Pull request merged**